3-Solving-Problems-By-Searching/poisson_distribution.js

```diff
@@ -0,0 +1,78 @@
+function poissonDistribution(height, width, radius) {
+  let sample = poissonDiscSampler(width, height, radius);
+  let nodes = [];
+  let s = sample();
+  let grid = [];
+  while (s) {
+    nodes.push(s.point);
+    grid = s.grid;
+    s = sample();
+  }
+  return [nodes, grid];
+}
+
+function poissonDiscSampler(width, height, radius) {
```

Comment:
I had no idea there are so much restrictions to even open source code. This rules make sense though. Thanks for the information ! 
I will either generate a JSON using the poisson-disk-sampling library or try the gittered grid.  Whichever works better, we can use.